app/scenes/channel_info/channel_info_row.js

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PropTypes} from 'react';
+import {
+    StyleSheet,
+    Switch,
+    Text,
+    TouchableHighlight,
+    View
+} from 'react-native';
+import Icon from 'react-native-vector-icons/FontAwesome';
+
+import FormattedText from 'app/components/formatted_text';
+
+const style = StyleSheet.create({
+    container: {
+        backgroundColor: '#fff',
+        paddingHorizontal: 15,
+        flexDirection: 'row',
+        alignItems: 'center'
+    },
+    detail: {
+        marginHorizontal: 15,
+        color: 'rgba(0, 0, 0, 0.5)',
+        fontSize: 15
+    },
+    label: {
+        flex: 1,
+        marginLeft: 15,
+        fontSize: 15,
+        paddingVertical: 15
+    }
+});
+
+function createTouchableComponent(children, action) {
+    return (
+        <TouchableHighlight onPress={action}>
+            {children}
+        </TouchableHighlight>
+    );
+}
+
+function ChannelInfoRow(props) {
```

Comment:
I'm 0/5 with changing it, I just wanted to know the reason and that makes total sense to me, @hmhealey what are you're thoughts on this?